modules/storages/app/models/storages/storage.rb

```diff
@@ -89,7 +90,7 @@ class Storages::Storage < ApplicationRecord
   def self.shorten_provider_type(provider_type)
     case /Storages::(?'provider_name'.*)Storage/.match(provider_type)
     in provider_name:
-      provider_name.downcase
+      provider_name.underscore
     else
       raise ArgumentError,
             "Unknown provider_type! Given: #{provider_type}. " \
```

Comment:
🟡 I have this gut feeling that this method could be removed or at least made into a good old `if` as we only have 2 cases.

Reply:
we use it in a view, where we not yet have storage instances, but instead call in on the class. I'd keep it.